iceoryx2-bb/cxx/tests/CMakeLists.txt

```
@@ -0,0 +1,39 @@
+# Copyright (c) 2024 Contributors to the Eclipse Foundation
+#
+# See the NOTICE file(s) distributed with this work for additional
+# information regarding copyright ownership.
+#
+# This program and the accompanying materials are made available under the
+# terms of the Apache Software License 2.0 which is available at
+# https://www.apache.org/licenses/LICENSE-2.0, or the MIT license
+# which is available at https://opensource.org/licenses/MIT.
+#
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+
+cmake_minimum_required(VERSION 3.22)
+
+project(iceoryx2-bb-cxx-tests VERSION ${IOX2_VERSION_STRING} LANGUAGES CXX)
+
+include(cmake/googletest.cmake)
+
+find_package(iceoryx2-cxx REQUIRED)
```

I guess this is just a typo and you meant this
```suggestion
find_package(iceoryx2-bb REQUIRED)
```

---

Since we are only porting the containers to C++ I would call it `iceoryx2-bb-container-cxx` and add the `cxx` suffix at the end so that it is equal to `iceoryx2-bb-container` crate for the first part. It makes it clear that it is the C++ version of the crate/library.

If we every have other versions of that crate we can then follow the name scheme `iceoryx2-bb-container-LANGUAGE`

---

There is two possibilities here, always have a prefix `iceoryx2-LANG-...` or always have the language as a suffix `iceoryx2-...-LANG`.

The ffi targets seem to favor the prefix variant at the moment.

---

I prefer the suffix variant since then the closely related things are close together in a literal sort. Like in my IDE file explorer or with ls.